sw/airborne/firmwares/fixedwing/stabilization/stabilization_adaptive.c

```diff
@@ -489,6 +553,17 @@ inline static void h_ctl_pitch_loop(void)
   h_ctl_ref.pitch_accel = 0.;
 #endif
 
+#ifdef USE_KFF_UPDATE
```

Comment:
Are you really using this ? I did this a really long time ago but never really finished it...